docs/releases/5.2.txt

```diff
@@ -187,7 +187,8 @@ Logging
 Management Commands
 ~~~~~~~~~~~~~~~~~~~
 
-* ...
+* The :djadmin:`shell` now auto-imports models from your app. Furthermore
+  you can customize the behavior of the shell to import extra-things.
```

Comment:
```suggestion
* The :djadmin:`shell` management command now auto-imports all models from your
  :setting:`INSTALLED_APPS`. You can also :ref:`customize this behaviour
  <customizing-shell-auto-imports>` to prevent these auto-imports or to add
  additional imports.
```